### ISSUE TITLE
Scoped folders are incorrectly marked invalid under Windows O/S

### ISSUE THREAD

**Issue by AgentOren** [Semigradsky/read-installed#1]
Under Windows O/S, scoped folders (eg. `@prisma/client`) are falsely marked as invalid, due to:

https://github.com/Semigradsky/read-installed/blob/571e45fffc04b21f209fccc574a46899f9124b6f/read-installed.js#L201

in the above example, name exists but `@prisma/client` !== `@prisma\client` (notice the backslash difference).

Can be fixed by always converting to forward slashes, earlier in the code:
https://github.com/Semigradsky/read-installed/blob/571e45fffc04b21f209fccc574a46899f9124b6f/read-installed.js#L152

can be changed to:
```javascript
          installed = i
              .filter(function (f) {
                  return f.charAt(0) !== '.';
              })
              .map(function (f) {
                  return f.replace(/\\/g, '/');
              });
```

**Comment by Semigradsky** [comment]
Thanks!

**Comment by Semigradsky** [comment]
Fixed in c148815
